kentik_api_library/kentik_api/api_connection/api_connector.py

```diff
@@ -38,14 +53,38 @@ def send(self, api_call: APICall, payload: Optional[Dict[str, Any]] = None) -> A
         else:
             raise ValueError(f"Improper API call method: {api_call.method}")
 
-        self._validate_response(response)
+        # TODO(dfurman): consider catching and re-raising Requests exceptions
```

Comment:
sounds good; this way the implementation details of library dont leak out to the library public interface